op-batcher/batcher/config.go

```diff
@@ -124,6 +130,12 @@ func (c *CLIConfig) Check() error {
 	if c.Compressor == compressor.RatioKind && (c.ApproxComprRatio <= 0 || c.ApproxComprRatio > 1) {
 		return fmt.Errorf("invalid ApproxComprRatio %v for ratio compressor", c.ApproxComprRatio)
 	}
+	if c.CompressionAlgo == "" {
+		return errors.New("must set compression algo")
+	}
+	if (c.CompressionAlgo == "zlib" && (c.CompressLevel < 0 || c.CompressLevel > 9)) || (c.CompressionAlgo == "brotli" && (c.CompressLevel < 1 || c.CompressLevel > 11)) {
+		return fmt.Errorf("invalid compression level %v for %v", c.CompressLevel, c.CompressionAlgo)
+	}
```

Comment:
This should be encapsulated in helpers around the enums like `IsValidComressorAlgo(CompressorAlgo)`